pkg/api/types.go

```diff
@@ -2103,6 +2103,12 @@ type PodSpec struct {
 	// file if specified. This is only valid for non-hostNetwork pods.
 	// +optional
 	HostAliases []HostAlias
+	// If specified, indicates the pod's priority. "system" is a special keyword
+	// which indicates the highest priority. Any other name must be defined in
+	// Admission Controller config or pod will be rejected.
+	// If not specified, the pod priority will be zero.
```

Comment:
@smarterclayton This is a reference to a priority class object. The field is already named `PriorityClassName`.

@k82cn that's the plan.